stm32-modules/heater-shaker/firmware/system/system_hardware.h

```diff
@@ -1,16 +1,65 @@
 #ifndef SYSTEM_HARDWARE_H__
 #define SYSTEM_HARDWARE_H__
 #include "stm32f3xx_hal_gpio.h"
+#include "stm32f3xx_hal.h"
 #ifdef __cplusplus
 extern "C" {
 #endif  // __cplusplus
 
+#include <stdbool.h>
+#include "systemwide.h"
+
+#define I2C_ADDRESS        0x6C
```

Comment:
these values are only used in `system_hardware.c`, can we have them at the beginning of that file? then we can keep this file for just things that need to be shared between `system_hardware.c` and `freertos_system_task.cpp`